sdk/energyservices/ci.yml

```diff
@@ -0,0 +1,32 @@
+ # NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+
+trigger:
+  branches:
+    include:
+    - main
+    - hotfix/*
+    - release/*
+  paths:
+    include:
+    - sdk/energyservices/
+
+pr:
+  branches:
+    include:
+    - main
+    - feature/*
+    - hotfix/*
+    - release/*
+  paths:
+    include:
+    - sdk/energyservices/
+    - sdk/resourcemanager/
```

Comment:
@ArthurMa1978 @m-nash After the GA and the update in https://github.com/Azure/azure-sdk-for-net/pull/28064 to a PackageReference I assume we no longer want these triggers for sdk/resourcemanager in the individual resource manager package pipelines. Can these now be cleaned up?

Reply:
Yes, will fix it soon

Reply:
Since @arthur has an action item, resolving the thread.